src/precompile.jl

```diff
@@ -1,13 +1,19 @@
 using PrecompileTools
 
 @setup_workload begin
+    # Maximum number of steps
+    maxsteps = 2
     # Starting time of integration
     jd0 = datetime2julian(DateTime(2000,1,1,12)) 
     # Number of years 
     nyears = 2031.0 - year(julian2datetime(jd0))
+    # Dense output
+    dense = Val(true)
+    # Dynamical function 
+    dynamics = DE430!
```

Comment:
Should these constants be `local`s?

Reply:
Actually, they are treated as globals, so I think I should add `const`.